pkg/webhook/policy.go

```diff
@@ -243,33 +210,25 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 					logging.Process, "admission",
 					logging.EventType, "violation",
 					logging.ConstraintName, r.Constraint.GetName(),
-					logging.ConstraintGroup, r.Constraint.GroupVersionKind().Group,
```

Comment:
Why are we removing group/version from the log lines and events?

Reply:
My bad (late night commits should be forbidden)

Reply:
LOL no worries, thought I missed something

Reply:
It looks like group/version are still missing?